### PR TITLE
Fix warnings that 'yarn start' compalins about so we don't accidentally add new ones.

### PR DESCRIPTION
**Pull request by mikelehen**

### CODE REVIEW

File: src/components/CountyMap/CountyMap.js [hunk 1]
```diff
@@ -33,23 +32,22 @@ const CountyMap = ({
...
-      '#ffcec5',
-      '#ffad9f',
-      '#ff8a75',
-      '#ff5533',
-      '#e2492d',
-      '#be3d26',
-      '#9a311f',
-      '#782618',
-    ]);
+  // TODO: Add fill back (see comment below)
```
Comment (mikelehen) [inline src/components/CountyMap/CountyMap.js:35]: I have no context on this stuff. If you know if it's realistically dead, I can just delete it.

**Comment by zjrosen1** [comment]
i cleaned up some these in my pr, lets hold off for a sec on this one

**Comment by mikelehen** [comment]
Okaydoke!

**Comment by mikelehen** [comment]
Closing for now.  Can reopen if it ends up being useful.

**Pull request closed without merging**